src/nvim/generators/gen_api_ui_events.lua

```diff
@@ -205,7 +205,8 @@ for _, ev in ipairs(events) do
     ev_exported[attr] = ev[attr]
   end
   for _, p in ipairs(ev_exported.parameters) do
-    if p[1] == 'HlAttrs' then
+    if p[1] == 'HlAttrs' or p[1] == 'Dict' then
+      -- TODO(justinmk): for back-compat, but do clients actually look at this?
       p[1] = 'Dictionary'
     end
```

Comment:
For now I kept the ui_events parameter type the same, since it's possible that clients use that(?).